docs/dev/table/tableApi.md

```diff
@@ -2882,13 +2922,13 @@ val result = orders
       <td>
         <p>Groups and aggregates a table on a <a href="#group-windows">group window</a> and possibly one or more grouping keys. You have to close the "flatAggregate" with a select statement. And the select statement does not support aggregate functions.</p>
 {% highlight scala %}
-val tableAggFunc = new MyMinMax
+val top2 = new Top2()
```

Comment:
`Top2()` -> `Top2` is enough for scala. But I am fine if want keeps the current style.

Reply:
Make sense. I will remove the brackets to make it more scala style.